backend/models/index.js

```diff
@@ -1,25 +1,34 @@
-const dbConfig = require("../config/db.config.js");
-const Sequelize = require("sequelize");
-
-// CREATE SEQUELIZE INSTANCE
-const db = {};
-
-// DEFINE SEQUALIZE INSTANCE
-// SEQUELIZE- Option 3: Passing parameters separately (other dialects)
-const sequelize = new Sequelize(dbConfig.DB, dbConfig.USER, dbConfig.PASSWORD, {
-	host: dbConfig.HOST,
-	dialect: dbConfig.dialect,
-
-	pool: {
-		max: dbConfig.pool.max,
-		min: dbConfig.pool.min,
-		acquire: dbConfig.pool.acquire,
-		idle: dbConfig.pool.idle,
-	},
-});
-
-// RETURN SEQUELIZE INSTANCE
-db.sequelize = sequelize;
-db.Sequelize = Sequelize;
-
+const {Sequelize, DataTypes} = require('sequelize');
+
+//connecting to database
+const sequelize = new Sequelize('testdb', 'admin', 'admin123',{

```

Comment:
suggesting to revert to seperate db config, avoiding redundancy to change dbconfig in multiple files in the future.
```suggestion
const sequelize = new Sequelize(dbConfig.DB, dbConfig.USER, dbConfig.PASSWORD, {
	host: dbConfig.HOST,
	dialect: dbConfig.dialect,

	pool: {
		max: dbConfig.pool.max,
		min: dbConfig.pool.min,
		acquire: dbConfig.pool.acquire,
		idle: dbConfig.pool.idle,
	},
});
```

Reply:
suggesting to revert to seperate db config, avoiding redundancy to change dbconfig in multiple files in the future.
```suggestion
const sequelize = new Sequelize(dbConfig.DB, dbConfig.USER, dbConfig.PASSWORD, {
	host: dbConfig.HOST,
	dialect: dbConfig.dialect,

	pool: {
		max: dbConfig.pool.max,
		min: dbConfig.pool.min,
		acquire: dbConfig.pool.acquire,
		idle: dbConfig.pool.idle,
	},
});
```